pkg/services/sqlstore/migrations/short_url_mig.go

```diff
@@ -0,0 +1,18 @@
+package migrations
+
+import . "github.com/grafana/grafana/pkg/services/sqlstore/migrator"
+
+func addShortUrlMigrations(mg *Migrator) {
+	shortUrlV1 := Table{
+		Name: "short_url",
+		Columns: []*Column{
+			{Name: "uid", Type: DB_NVarchar, Length: 40, Nullable: false, IsPrimaryKey: true},
+			{Name: "path", Type: DB_Text, Nullable: false},
+			{Name: "created_by", Type: DB_Int, Nullable: false},
+			{Name: "created_at", Type: DB_DateTime, Nullable: false},
+			{Name: "last_seen_at", Type: DB_DateTime, Nullable: true},
```

Comment:
Please use DB_Int as types for date. Not all databases like SQLite store date with timezone information why we suggest storing epochs, like can be seen [here](https://github.com/grafana/grafana/blob/ffc11cb63c725c0eecc6154657ad1f3d07e0030f/pkg/services/sqlstore/migrations/user_auth_token_mig.go#L20-L21) for example.

However, it has the drawback that the epoch fields have to be set manually, like can be seen [here](https://github.com/grafana/grafana/blob/ffc11cb63c725c0eecc6154657ad1f3d07e0030f/pkg/services/auth/auth_token.go#L78-L79).